demos-europe/demosplan-core
# bug(refs T32727): Add storage key for pagination

**Ticket:** https://yaits.demos-deutschland.de/T32727

Add session storage key for currentPage and items per page for list statements and list segments.

## How to test

Go to statement list or segments list and change page and/or items per page. Go to another page and back to the list. Page and items per page should be the same.
